src/System.IO.Compression/src/System/IO/Compression/DeflateStream.cs

```diff
@@ -209,6 +209,14 @@ public override void Flush()
             return;
```

Comment:
Makes sense. If that's what we think it should do, I can fix that for both Flush and FlushAsync.  @ericstj?